src/gui/servernotificationhandler.cpp

```diff
@@ -49,24 +49,24 @@ void ServerNotificationHandler::slotFetchNotifications(AccountStatePtr ptr)
     }
 
     // if the previous notification job has finished, start next.
-    _notificationJob = new JsonApiJob(ptr->account(), notificationsPath, this);
-    QObject::connect(_notificationJob.data(), &JsonApiJob::jsonReceived,
-        this, &ServerNotificationHandler::slotNotificationsReceived);
-    _notificationJob->setProperty("AccountStatePtr", QVariant::fromValue<AccountStatePtr>(ptr));
+    auto *job = new JsonApiJob(ptr->account(), notificationsPath, {}, {}, this);
+    QObject::connect(job, &JsonApiJob::finishedSignal,
+        this, [job, ptr, this] {
```

Comment:
Code style wise, would it make sense to include empty parentheses when defining a lambda?

Reply:
I think clang even warns when doing so.

Reply:
I've included them in all lambdas in my upcoming PR, and I haven't seen any warnings. I think it's really just a matter of preference.